samples/Makefile

```diff
@@ -64,7 +64,7 @@ rand_pfault_check: rand_pfault_check.c compdecomp_th.c ../lib/libnxz.a
 	rand_pfault_check.c compdecomp_th.c $(LDLIB)
 
 nx_gzip: nx_gzip.c
-	$(CC) $(CFLAGS) -o $@ $^ -lz
+	$(CC) $(CFLAGS) -I../inc_nx -I../lib $(LDFLAGS) -L/usr/lib/ -o $@ $^ $(LDLIB)
```

Comment:
You don't need `-I../lib` or `-L/usr/lib/` here, this should work:
`$(CC) $(CFLAGS) -I$(INC) $(LDFLAGS) -o $@ $^ $(LDLIB)`

Reply:
Fixed!